<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UE Plugin Builder</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_1">
      <item>
       <widget class="QPushButton" name="btnSelectUEPath">
        <property name="text">
         <string>Select UE Path</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editUEPath"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btnSelectPluginFile">
        <property name="text">
         <string>Select .uplugin File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editPluginFile"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="btnSelectPackageFolder">
        <property name="text">
         <string>Select Package Folder</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editPackageFolder"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btnBuild">
      <property name="text">
       <string>Build Plugin</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="consoleOutput">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
